src/containers/profile/index.jsx

```diff
@@ -32,17 +33,17 @@ export const Profile = () => {
 
   return (
     <section className={profile}>
```

Comment:
There is a good practise, to mark tags `<article>` `<section>` with `<h1>...<h6>.`
In `<h>` you should write description of the block and then use css property display: none if this `<h> ` isn't needed in the design of the project